### PR TITLE
Added kuzzle-common-objects

### PR DESCRIPTION
**Pull request by jenow**

### CODE REVIEW

**Comment by stafyniaksacha** [comment]
+1

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "bufferutil": "^1.2.1",
     "cli-color": "^1.1.0",
     "easy-circular-list": "^1.0.13",
+    "kuzzle-common-objects": "git+https://github.com/kuzzleio/kuzzle-common-objects.git",
```
Comment (benoitvidis) [inline package.json]: Same remark as @ballinette on Kuzzle side. Shouldn't we use a tag?

**Comment by benoitvidis** [comment]
+1

**Comment by ballinette** [comment]
+1

**Pull request merged**